docs/general/manage-tokens/oasis-wallets/web.mdx

```diff
@@ -0,0 +1,390 @@
+mnemonic based on the [ADR-8 derivation scheme][adr8]. Select one or more
+accounts and click the "Open" button to import them into your wallet.
+
+![Account derivation popup](../../images/wallet/web/create_new_wallet_select_accounts.png)
+
+### User Profile
+
+If you want to permanently store the keys of selected accounts, select the
+"Store private keys locally, protected by a password" box. After entering a
+password below, this will **instantiate a profile inside the local store of your
+browser to safely store your keys**. To access them, you will need to enter the correct
+password each time you will open the Oasis Wallet - Web.
+
+Finally, you will be taken to the *Wallet screen*, containing
+information about your account balance, recent transactions and more.
+
+![The Wallet screen](../../images/wallet/web/wallet.png)
+
+[adr8]: ../../../adrs/0008-standard-account-key-generation.md
+
+## Import an Existing Account
+
+On the "Open wallet" page select whether you want to open your
+wallet via a mnemonic, a private key, or a Ledger hardware wallet.
+
+![Access an Existing Wallet](../../images/wallet/web/import_wallet.png)
+
+<Tabs>
+
+<TabItem value="Mnemonic">
+
+In the "Enter your keyphrase here" field, enter each word of your mnemonic
+separated by a space. Then, hit the "Import my wallet" button.
+
+![Open Wallet via Mnemonic](../../images/wallet/web/import_wallet_mnemonic.png)
+
+The [account derivation popup](#account-derivation) will be shown where
+you can pick one or more derived accounts to import.
+
+:::info
+
+Oasis Wallet - Web uses English mnemonic phrase words as defined in
+[BIP39](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki).
+You can find a complete list of all valid phrase words
+[here](https://github.com/bitcoin/bips/blob/master/bip-0039/english.txt).
+If you misspelled a word, the wallet will warn you.
+
+:::
+
+</TabItem>
+
+<TabItem value="Private Key">
+
+Paste your Base64-encoded Ed25519 private key in the "Enter your private key
+here" field and then click on the "Import my wallet" button.
+
+![Open Wallet via Private Key](../../images/wallet/web/import_wallet_private_key.png)
+
+Selecting "Store private keys locally, protected by a password" will instantiate
+the [user's profile](#user-profile).
+
+:::tip
+
+The Ed25519 private key is 64 bytes long and consists of two 32-byte parts:
+
+1. The first 32 bytes are the **private key** used for signing the transactions
+   (e.g. for sending tokens from your account).
+2. The last 32 bytes are the **public key** used to verify the signatures
+   of your transactions and also derive your account's address of the form
+   `oasis1...`.
+
+:::
+
+:::danger
+
+If you typed in the private key manually, make sure you **do not mix similar
+characters** like the big O and 0 or big I and 1!
+
+Currently, there is no error detection on the inputted private key, so the
+wallet cannot warn you of a mistyped character. **It is imperative that you
```

Comment:
Extension doesn't check but on web wallet:
- if you typo in a private key that contains public part too: https://github.com/oasisprotocol/oasis-wallet-web/pull/1007
- if you typo in a private key that does not contain public part: it will open an account with a different address